src/Resource.php

```diff
@@ -20,9 +20,9 @@ public function getApiKey()
         return $this->parent->getApiKey();
     }
 
-    public function getUri()
+    public function getUri($subcommand = false)
     {
-        return $this->parent->getUri() . $this->id . '/';
+        return $this->parent->getUri(true) . $this->id . ($subcommand?'/':'');
```

Comment:
hey, thanks for first pull request :)
I think that better solution would be something like
`return $this->parent->getUri() . '/' . $this->id;`
and removing trailing / from `Api::API_URI`
What do you think?